k2mosaic/mosaic.py

```diff
@@ -115,7 +115,7 @@ def add_pixels(self, tpf):
         idx = self.cadenceno - tpfdata["CADENCENO"][0]
 
         # When time is nan, we know that there is no available data.
```

Comment:
One minor suggestion: let's update the comment to explain the new statement, e.g.:
```suggestion
        # When quality flag 65536 is raised, there is no data and the times are NaN.
```